src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -0,0 +1,645 @@
+  </data>
+  <data name="Globals_Rendering.Text" xml:space="preserve">
+    <value>Rendering</value>
+  </data>
+  <data name="Globals_ShowTitlebar.Content" xml:space="preserve">
+    <value>Show the title bar</value>
+  </data>
+  <data name="Globals_ShowTitlebar.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>When checked, the tabs are moved into the title bar and the title bar disappears. When unchecked, the title bar sits above the tabs.</value>
+  </data>
+  <data name="Globals_ShowTitleInTitlebar.Content" xml:space="preserve">
+    <value>Show terminal title in title bar</value>
+  </data>
+  <data name="Globals_ShowTitleInTitlebar.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>When checked, the title bar displays the title of the selected tab. When unchecked, the title bar displays 'Windows Terminal'.</value>
+  </data>
+  <data name="Globals_SnapToGridOnResize.Content" xml:space="preserve">
+    <value>Window resize behavior</value>
+  </data>
+  <data name="Globals_SnapToGridOnResize.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>When checked, the window will snap to the nearest character boundary on resize. When unchecked, the window will resize smoothly.</value>
+  </data>
+  <data name="Globals_SoftwareRendering.Content" xml:space="preserve">
+    <value>Software rendering</value>
+  </data>
+  <data name="Globals_SoftwareRendering.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>When checked, the terminal will use the software renderer (a.k.a. WARP) instead of the hardware one.</value>
+  </data>
+  <data name="Globals_StartOnUserLogin.Content" xml:space="preserve">
+    <value>Launch on machine startup</value>
+  </data>
+  <data name="Globals_StartOnUserLogin.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>When checked, this enables the launch of Windows Terminal at machine startup.</value>
+  </data>
+  <data name="Globals_Startup.Text" xml:space="preserve">
+    <value>Startup</value>
+  </data>
+  <data name="Globals_TabWidthMode.Header" xml:space="preserve">
+    <value>Tab width mode</value>
+  </data>
+  <data name="Globals_TabWidthMode.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Sets the width of the tabs. 'Equal' sizes each tab to the same width. 'Title length' sizes each tab to the length of its title. 'Compact' sizes each tab to the length of its title when focused, and shrinks to the size of only the icon when the tab is unfocused.</value>
+  </data>
+  <data name="Globals_Theme.Header" xml:space="preserve">
+    <value>Theme</value>
+  </data>
+  <data name="Globals_Theme.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Sets the theme of the application. The value 'system' refers to the active Windows system theme.</value>

```

Comment:
Actually, if you can, just remove the 'system' sentence and put that on the 'system' value.

Reply:
Couple things:
- I changed this to 'default' since 'system' isn't an option on the UI
- You can't add tooltips to specific radio button items, so I'll add a comment similar to the previous ones about replacing the text with the Uid value